aria-practices.html

```diff
@@ -4869,6 +4887,11 @@ <h4><span id="naming_role_guidance_heading">Accessible Name Guidance by Role</sp
                 </ul>
               </td>
             </tr>
+            <tr>
+              <td><a href="#time" class="role-reference"><code>time</code></a></td>
+              <td>Do Not Name</td>
+              <td>Not supported by assistive technologies.</td>
```

Comment:
Consider changing:
> Not supported by assistive technologies.

to:
> Naming not supported by assistive technologies.

... just to ensure that authors don't accidentally get the impression that the _time role_ itself is not supported by AT...

Reply:
Thanks, fixed.